src/models/LicenseList.ts

```diff
@@ -19,6 +19,22 @@ export default class LicenseList {
     return Object.values(this.list)
   }
 
+  getUniqueList(): License[] {
```

Comment:
I think this pattern match can not determine uniqueness.
`object-assign` and `object.assign` are different libraries.
Different libraries should be listed even If these have the same functionality.

Reply:
Ah good point, I assumed they were the same. How should they be added to the list if they are converted to the same key?

Reply:
It have been already [made unique list]( https://github.com/k-tomoyasu/react-native-oss-license/blob/master/src/reader/DependencyReader.ts ), so you don't have to think about that here.  
They are added each to list If they are not exactly the same key.

Reply:
I have removed the unique list function 👍